src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js

```diff
@@ -14,14 +14,15 @@ const propTypes = {
 const defaultProps = {};
 
 const names = {
-  barrierfree: 'ch.sbb.barrierfreierbahnhoefe',
+  barrierfree: '',
   notBarrierfree: 'ch.sbb.nichtbarrierfreierbahnhoefe',
   stuetzpunkt: 'bahnhof_plural',
 };
 
 const descriptions = {
   de: {
-    barrierfree: ' an denen autonomes Reisen möglich ist.',
+    barrierfree:
+      'Ganz oder hauptsächlich barrierefreie Bahnhöfen an denen das autonome Reisen möglich ist.',
```

Comment:
Replace
`Nicht barrierefreie Bahnhöfe an denen automes Reisen nicht möglich ist.`
by
`Nicht barrierefreie Bahnhöfe, an denen autonomes Reisen nicht möglich ist.`

Add comma as well as `autonomes`.

Reply:
But we do not write:
'Nicht barrierefreie Bahnhöfe an denen automes Reisen nicht möglich ist.'
but 
'Ganz oder hauptsächlich barrierefreie Bahnhöfen an denen das autonome Reisen möglich ist.'

Reply:
but here https://github.com/geops/trafimage-maps/blob/a914108897c6052561a09593b2ff3cbd3f2ecb61/src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js#L26, see suggestion below